### ISSUE TITLE
Create PDF views via API from importer (was: PDF views not automatically created)

### ISSUE THREAD

**Issue by moman822** [NRGI/resourcedata.org#178]
We don't have views for the PDF documents created automatically. I was able to set one up via the CMS but can we do this for all documents?

![image](https://user-images.githubusercontent.com/19787025/29634258-38b7ddae-8817-11e7-9592-65f98ef84c8b.png)

**Comment by mattfullerton** [comment]
Is this a general problem? I fear it is.

The problem is described in detail in #45 

Now the problem we have is we concluded:
- This isn't a problem for RGI because we do a one time import and I intended to make resources update not replace (#45)
- Then later I concluded I didn't need to handle the resources carefully for RGI because we were doing a one time import (#88)
- And then we changed the schema and had to redo the RGI import ...d'oh

The underlying problem is that CKAN doesn't do automatic view creations when the dataset isn't new (or something to that effect), so I think the only bullet proof way to fix this is to have the import create the views.

Now would also be a good time to decide if its crucial that resource links for RGI _never_ change for the occasional times that we might do an update (i.e. schema change, question notation or whatever).
